NitroxClient/MonoBehaviours/Gui/Chat/PlayerChatInputField.cs

```diff
@@ -82,9 +82,8 @@ private void Update()
 
             if (selected)
             {
-                if (!string.IsNullOrEmpty(InputField.text))
+                if (!string.IsNullOrEmpty(InputField.text) && !string.IsNullOrWhiteSpace(InputField.text))
```

Comment:
```suggestion
                if (!string.IsNullOrWhiteSpace(InputField.text))
```

Reply:
Oh, lol. Yeah, that would make sense. Dumb me.  :D
Just came home from work, I see you went ahead and fixed that oversight. Thank you! :)